<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>534</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DengueME</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="editorView" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="modelFile">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="margin">
            <number>6</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDefault">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Default Values</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonHelp">
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="icon">
            <iconset resource="Resource.qrc">
             <normaloff>:/img/Resources/help.png</normaloff>:/img/Resources/help.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonClose">
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="icon">
            <iconset resource="Resource.qrc">
             <normaloff>:/Resources/close.png</normaloff>:/Resources/close.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Editor" name="editor" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewModel"/>
    <addaction name="actionNewProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionRename"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionRunByStep"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionModelBuilder"/>
    <addaction name="actionSetWorkspace"/>
    <addaction name="actionSynchronize"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionResetViews"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionNewModel"/>
   <addaction name="actionSave"/>
   <addaction name="actionRename"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionBuilder"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionRunByStep"/>
  </widget>
  <widget class="QDockWidget" name="explorerDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Projects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="WorkspaceView" name="treeView">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="outputDock">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="outputBrowser"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear Console</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="run_stopButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>Resources/about.png</normaloff>Resources/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/exit.png</normaloff>:/img/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/save.png</normaloff>:/img/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/run.png</normaloff>:/img/Resources/run.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>F9, Ctrl+R</string>
   </property>
  </action>
  <action name="actionProject">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/folder-new.png</normaloff>:/img/Resources/folder-new.png</iconset>
   </property>
   <property name="text">
    <string>New &amp;project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEpidemiological">
   <property name="text">
    <string>E&amp;pidemiological</string>
   </property>
  </action>
  <action name="actionCoupled">
   <property name="text">
    <string>&amp;Coupled</string>
   </property>
  </action>
  <action name="actionIntegrated">
   <property name="text">
    <string>&amp;Integrated</string>
   </property>
  </action>
  <action name="actionTemporal">
   <property name="text">
    <string>&amp;Temporal</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/configure.png</normaloff>:/img/Resources/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resources/close.png</normaloff>:/Resources/close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close model </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRunByStep">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/run-step.png</normaloff>:/img/Resources/run-step.png</iconset>
   </property>
   <property name="text">
    <string>Run &amp;step by step</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionSetWorkspace">
   <property name="text">
    <string>Change &amp;workspace</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/rename.png</normaloff>:/img/Resources/rename.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/delete.png</normaloff>:/img/Resources/delete.png</iconset>
   </property>
   <property name="text">
    <string>R&amp;emove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionNewModel">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/model-new.png</normaloff>:/img/Resources/model-new.png</iconset>
   </property>
   <property name="text">
    <string>New &amp;model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionModelBuilder">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/builderTool.png</normaloff>:/img/Resources/builderTool.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Model builder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionResetViews">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/Synchronize-64.png</normaloff>:/img/Resources/Synchronize-64.png</iconset>
   </property>
   <property name="text">
    <string>Reset views</string>
   </property>
  </action>
  <action name="actionBuilder">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/img/Resources/builderTool.png</normaloff>:/img/Resources/builderTool.png</iconset>
   </property>
   <property name="text">
    <string>Model Builder</string>
   </property>
  </action>
  <action name="actionSynchronize">
   <property name="text">
    <string>Synchronize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionOpenExplorer">
   <property name="text">
    <string>Open in Explorer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WorkspaceView</class>
   <extends>QTreeView</extends>
   <header location="global">workspaceview.h</header>
  </customwidget>
  <customwidget>
   <class>Editor</class>
   <extends>QWidget</extends>
   <header>editor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
